proposals/lip-0051.md

```diff
@@ -202,8 +202,6 @@ We further use the utility function `getMainchainID()` defined in [LIP 0037][lip
 | `Module ` | string | Must be of length at least `MIN_MODULE_NAME_LENGTH` and at most `MAX_MODULE_NAME_LENGTH`.| Used for identifying modules.|
 | `TokenID` | bytes | Must be of length `TOKEN_ID_LENGTH`. | Used for token identifiers. |
 | `ChainID` | bytes | Must be of length `CHAIN_ID_LENGTH`. | Used for chain identifiers. |
-| `Transaction` | object | Must follow the `transactionSchema` schema defined in the [LIP 0068][lip-0068] with the only difference that `params` property is not serialized and contains the values of parameters of `paramsSchema` for the corresponding transaction. | An object representing a non-serialized transaction. |
-| `CCM` | object | Must follow the [`crossChainMessageSchema` schema defined in LIP 0049][lip-0049#ccmschema]. | The type of cross-chain messages. |
```

Comment:
Also here we my consider to keep the definition

Reply:
As we just have one type `CCM` that is the same everywhere and I don't see any risk of confusing about it, I would prefer to avoid this redundancy in LIPs.